ra2ce/graph/networks.py

```diff
@@ -21,9 +21,12 @@
 
 import logging
 from pathlib import Path
-
+import geopandas as gpd
```

Comment:
please have a look at which imports you have, preferably use one or the other not both